inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -187,7 +190,21 @@ public final class Constants {
                     .noDefaultValue()
                     .withDescription(
                             "The format of multiple sink, it represents the real format of the raw binary data");
-
+    public static final ConfigOption<String> PATTERN_PARTITION_MAP =
```

Comment:
Maybe 'sink.multiple.partition-pattern' can be reused here?

Reply:
sink.multiple.partition-pattern is not used and should be removed. @yunqingmoswu